### ISSUE TITLE
[Bug]: Exception on web, vibration package is broken

### ISSUE THREAD

**Issue by sgehrman** [hm21/pro_image_editor#139]
### Package Version

4.0.8

### Flutter Version

3.22

### Platforms

Web

### How to reproduce?

![Screenshot-20240625205458-1878x1590](https://github.com/hm21/pro_image_editor/assets/287137/a3f03d2b-3c26-4fd5-8353-d885d9cade0d)

### Example code (optional)

```Dart
set the debugger to break on all exceptions
```

### Device Model (optional)

linux

**Comment by sgehrman** [comment]
check out the pubspec.  It seems web is commented out?

https://github.com/benjamindean/flutter_vibration/blob/master/vibration/pubspec.yaml

https://github.com/benjamindean/flutter_vibration/issues/103

**Comment by sgehrman** [comment]
I added a lame hack that seems to work

```
    if (!kIsWeb) {
      Vibration.hasVibrator().then(
          (value) => layerInteractionManager.deviceCanVibrate = value ?? false);
      Vibration.hasCustomVibrationsSupport().then((value) =>
          layerInteractionManager.deviceCanCustomVibrate = value ?? false);
    }
```

**Comment by hm21** [comment]
Okay, good that you open a new issue in the vibration package because the issue is from that package and not from my package.
However, I released version `4.1.1` which also doesn't check `Vibration.hasVibrator()` when you disable `hitVibration` inside the helperline configs like below.

```dart
configs: ProImageEditorConfigs(
      helperLines: const HelperLines(
        hitVibration: !kIsWeb,
      ),
)
```

**Comment by sgehrman** [comment]
ok, thanks.
